File: examples/python/solana/create_account.py
# Usage: uv run python solana/create_account.py

import asyncio

from cdp import CdpClient
from dotenv import load_dotenv

load_dotenv()


async def main():
    async with CdpClient() as cdp:
        account = await cdp.solana.create_account()
        print(f"Successfully created account: {account.address}")


asyncio.run(main())

File: examples/typescript/solana/createAccount.ts
// Usage: pnpm tsx solana/createAccount.ts

import { CdpClient } from "@coinbase/cdp-sdk";
import "dotenv/config";

const cdp = new CdpClient();

const account = await cdp.solana.createAccount();
console.log(
  "Successfully created Solana account:",
  JSON.stringify(account, null, 2)
);
